tcl/pd-gui.tcl

```diff
@@ -362,6 +362,11 @@ proc init_for_platform {} {
             set ::cursor_editmode_resize "sb_h_double_arrow"
         }
         "win32" {
+            # Adding dejavu fonts on mswindows
+            package require twapi
+            twapi::AddFontResourceEx lib/DejaVuSansMono.ttf
+            twapi::AddFontResourceEx lib/DejaVuSansMono-Bold.ttf
+            #
```

Comment:
this should be protected by a `catch` clause.

if `twapi` isn't there (for whatever reasons), Pd is still perfectly usable, so there's little use in erroring out.

Reply:
My tcl is just trial and error, I'm not a programer. Could you do it? or tell me how to?

Anyway I have to commit a missing file on the "pdprototype.tgz" which is causing a Windows crash.

"pkgIndex.tcl"

I just did a build of this PR and got the error.

: - )